### PR TITLE
Update 2e2 to allow region changes at any moment

### PR DESCRIPTION
**Pull request by Nanush7**
Allow competitors to change their region of representation at any point during the year, but only once a year.

### CODE REVIEW

File: wca-regulations.md [hunk 1]
```diff
@@ -73,7 +73,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
...
     - 2e1) The eligible countries are defined by the [WCA List of Recognized Countries](https://www.worldcubeassociation.org/regulations/countries/).
-    - 2e2) Competitors may change their region of representation at any time, up to once a year. Exception: if the competitor loses citizenship of the region which they were previously representing, they must change the region of representation before or at their next competition (see [Regulation 2c2](regulations:regulation:2c2)).
+    - 2e2) Competitors may request changes to their region of representation, which may only be done at least one year after the last change to their region of representation and before any competition they are registered for. Exception: if the competitor loses citizenship of the region which they were previously representing, they must change the region of representation before or at their next competition (see [Regulation 2c2](regulations:regulation:2c2)).
```
Comment (Nanush7) [inline wca-regulations.md]:
```suggestion
    - 2e2) Competitors with updates to their citizenship status may change their country of representation before or at their first competition of a calendar year.
        - 2e2a) If the competitor loses citizenship of the country which they were previously representing, they must change the country of representation before or at their next competition (see Regulation 2c2).
        - 2e2b) If the competitor has acquired a new citizenship within the calendar year, they may change their country of representation to their new citizenship at any point between competitions.
```

Suggested by Callum Goodyear.

**Review by Rouxles — COMMENTED** [review]
Does this allow for region changes at any moment? Seems pretty unclear to me atm

**Comment by Nanush7** [comment]
The current wording was discussed with the WRT.

**Review by Nevseros — APPROVED** [review]
LGTM!

**Pull request merged**